integration-tests/it-producer/src/main/java/org/apache/servicecomb/it/schema/DownloadSchema.java

```diff
@@ -183,7 +185,7 @@ public String getFilename() {
       @ApiResponse(code = 200, response = File.class, message = ""),
   })
   public ResponseEntity<InputStream> netInputStream(String content) throws IOException {
-    URL url = new URL("http://localhost:9000/download/netInputStream?content="
+    URL url = new URL("http://localhost:" + netStreamPort + "/download/netInputStream?content="
```

Comment:
conflict with https://github.com/apache/incubator-servicecomb-java-chassis/pull/946/commits/31b9925a3c45c340489096ccf7d045e8cde755e5

copy that content to this commit or remote this commit, :)

Reply:
This commit has been removed. :)